packages/sdk-accounts/src/identity/sdk-types.ts

```diff
@@ -0,0 +1,30 @@
+import { Binary, SS58String } from "polkadot-api"
+import {
+  IdentityInfo as DescriptorsIdentityInfo,
+  IdentityJudgement,
+} from "./descriptors"
+
+export type IdentityInfo = {
+  [K in keyof DescriptorsIdentityInfo]?: DescriptorsIdentityInfo[K] extends
+    | Binary
+    | undefined
+    ? DescriptorsIdentityInfo[K]
+    : string
+}
```

Comment:
```suggestion
export type IdentityInfo = {
  [K in keyof DescriptorsIdentityInfo]?: K extends "pgp_fingerprint"
    ? DescriptorsIdentityInfo[K]
    : string
}
```
I wouldn't make it super-generic, b/c it's very coupled that that specific field... What I mean is that since we are in control of the `DescriptorsIdentityInfo` type, we may as-well do it like this, no?

Reply:
Agree 👍